src/test/java/study/SetTest.java

```diff
@@ -0,0 +1,48 @@
+package study;
+
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.assertj.core.api.Assertions.*;
+
+public class SetTest {
+    private Set<Integer> numbers;
+
+    @BeforeEach
+    void setUp() {
+        numbers = new HashSet<>();
+        numbers.add(1);
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+
+    @DisplayName("Set의 size() 메소드를 활용해 Set의 크기를 확인한다.")
+    @Test
+    void size() {
+        assertThat(numbers.size()).isEqualTo(3);
+    }
+
+    @DisplayName("Set의 contains() 메소드를 활용해 1, 2, 3의 값이 존재하는지 확인한다.")
+    @ParameterizedTest
+    @ValueSource(ints = {1, 2, 3})
+    void contains(int input) {
+        assertThat(numbers.contains(input)).isTrue();
```

Comment:
`StringTest` 를 보니 이미 잘 알고 계신 것 같지만, assertj 에서 제공하는 `contains` 로 변경해도 좋을 것 같네요!

```suggestion
        assertThat(numbers).contains(input);
```